LoopKitUI/View Controllers/AddEditOverrideTableViewController.swift

```diff
@@ -112,7 +112,11 @@ public final class AddEditOverrideTableViewController: UITableViewController {
     }
 
     private func configure(with settings: TemporaryScheduleOverrideSettings) {
-        targetRange = settings.targetRange
+        if let targetRange = settings.targetRange {
+            self.targetRange = DoubleRange(minValue: targetRange.lowerBound.doubleValue(for: glucoseUnit), maxValue: targetRange.upperBound.doubleValue(for: glucoseUnit))
+        } else {
+            targetRange = nil
```

Comment:
[nit] `self.targetRange = nil` might be a bit more readable (inline with 2 lines above).